receiver/udp.go

```diff
@@ -27,18 +27,25 @@ import (
 	"net"
 	"runtime"
 
+	"errors"
 	"github.com/telenornms/skogul"
+	"strconv"
+)
+
+const (
+	UDP_MAX_READ_SIZE = 65535
 )
 
 var udpLog = skogul.Logger("receiver", "udp")
 
 // UDP contains the configuration for the receiver
 type UDP struct {
-	Address string            `doc:"Address and port to listen to." example:"[::1]:3306"`
-	Handler skogul.HandlerRef `doc:"Handler used to parse, transform and send data."`
-	Backlog int               `doc:"Number of queued messages that are not delievered before the receiver starts blocking. Defaults to 100. Even when this is full, the kernel will still buffer data on top of this value. Higher values give smoother performance, but slightly more memory usage. Actual memory usage is a factor of average message size * backlog-fill."`
-	Threads int               `doc:"Number of worker go routines to use, which loosely translates to parallell execution. Defaults to number of CPU threads, with a minimum of 20. There is no correct number, but the value depends on how fast your senders are."`
-	ch      chan []byte       // Used to pass messages from the accept/read-loop to the worker pool/threads.
+	Address    string            `doc:"Address and port to listen to." example:"[::1]:3306"`
+	Handler    skogul.HandlerRef `doc:"Handler used to parse, transform and send data."`
+	Backlog    int               `doc:"Number of queued messages that are not delivered before the receiver starts blocking. Defaults to 100. Even when this is full, the kernel will still buffer data on top of this value. Higher values give smoother performance, but slightly more memory usage. Actual memory usage is a factor of average message size * backlog-fill."`
+	Threads    int               `doc:"Number of worker go routines to use, which loosely translates to parallel execution. Defaults to number of CPU threads, with a minimum of 20. There is no correct number, but the value depends on how fast your senders are."`
+	ch         chan []byte       // Used to pass messages from the accept/read-loop to the worker pool/threads.
+	PacketSize int               `doc:"UDP Packet size note: max. UDP read size is 65535"`
```

Comment:
Might be a nitpick, but I'd prefer if the private arguments are last, so if you swap the place of this and `ch`, that would be great. :)

Reply:
so all comments has to fixed with last commit